package.json

```diff
@@ -24,13 +24,12 @@
   "dependencies": {
     "broccoli-slow-trees": "^1.1.0",
     "debug": "^2.1.0",
-    "rsvp": "^3.0.14",
-    "sane": "^1.0.1"
+    "rsvp": "^3.0.18",
+    "sane": "^1.1.0"
```

Comment:
Can you update this to 1.1.1 Fixed a silly bug on my part
https://github.com/amasad/sane/commit/9aefcd7025fee3ca63f9f83ef161dfb53204cbf5